.storybook/addons.js

```diff
@@ -0,0 +1,2 @@
+import 'storybook-addon-jsx/register';
+import '@storybook/addon-actions/register';
```

Comment:
i thought this addon was for tracing events from actions?

Reply:
That's correct.  I can add that in a separate PR.

Reply:
yeah no problem.  I figure we were adding it.  It just made me searching in the PR for events. :)